sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -32,6 +32,8 @@ Set-SPOTenant
  [-AllowGuestUserShareToUsersNotInSiteCollection <Boolean>]
  [-AllowOverrideForBlockUserInfoVisibility]
  [-AllowSensitivityLabelOnRecords <Boolean>]
+ [-AllowSelectSecurityGroupsInSPSitesList [List<String>]]
```

Comment:
I realized that you are using List<String>. In PowerShell, the concept of an array of strings is represented by String[]. 
Could you update all these references accordingly?